Documentation/k8s-monitoring.md

```diff
@@ -5,40 +5,51 @@ Note that these steps work best with a local Kubernetes cluster running in `Vagr
 ## Prometheus Operator
 First the Prometheus operator needs to be started in the cluster so it can watch for our requests to start monitoring Rook and respond by deploying the correct Prometheus pods and configuration.
 A full explanation can be found in the [Prometheus operator repository on github](https://github.com/coreos/prometheus-operator), but the quick instructions can be found here:
-```
-wget -qO- https://raw.githubusercontent.com/coreos/prometheus-operator/master/deployment.yaml | kubectl create -f -
+```bash
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
```

Comment:
i'm thinking we should snap to a specific version (v0.8 i think) instead of using master.  That's what got us into this problem in the first place, because they changed `deployment.yaml` to `bundle.yaml` in master.  Do you have an opinion on if we should snap to an official release? That would probably be wise.